app/controllers/admin/defendants_court_applications_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Admin
+  class DefendantsCourtApplicationsController < Admin::ApplicationController
+    def index
+      @court_applications = if params[:query].present?
+                              CourtApplication.search(params[:query]).order(created_at: :desc).page(params[:page])
+                            else
+                              CourtApplication.order(created_at: :desc).page(params[:page])
```

Comment:
as I mentioned before,
you can `dry` the  `order(created_at: :desc).page(params[:page])`  since it is present in both if/else.